### PR TITLE
Chats

### PR DESCRIPTION
**Pull request by HenokSamuel**
Added components:
-Chat -displays all messages
-Message -message entry comp
-Chats - displays all chats
-ChatEntry -chat entry comp
-Added a redirect to createProfile route is no profile was made

### CODE REVIEW

**Review by Evtaft22 — DISMISSED** [review]

File: client/src/components/ChatEntry.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        >
+          <div>
+            <label>Chat Members:</label>
+            {participants.join(" ")}
+          </div>
+        </div>
+      </div>
+    </Link>
+  );
+};
```
Comment (jstrzesz) [inline client/src/components/ChatEntry.jsx:41]: add line after line 41

File: client/src/components/ChatEntry.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ChatEntry.propTypes = {
+  chatMessage: PropTypes.object.isRequired,
+};
+export default ChatEntry;
```
Comment (jstrzesz) [inline client/src/components/ChatEntry.jsx:47]: add line after line 45

File: client/src/components/ChatEntry.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            {participants.join(" ")}
+          </div>
+        </div>
+      </div>
+    </Link>
+  );
+};
+ChatEntry.propTypes = {
+  chatMessage: PropTypes.object.isRequired,
+};
```
Comment (jstrzesz) [inline client/src/components/ChatEntry.jsx:45]: add line after line 44

File: client/src/components/Chats.jsx [hunk 4]
```diff
@@ -1,2 +1,22 @@
...
+
+  return (
+    <div>
+      <h1>Current Messages (Inbox) </h1>
+      <ChatEntry chatMessage={chatMessage} />
+      <ChatEntry chatMessage={chatMessage} />
+      <ChatEntry chatMessage={chatMessage} />
+    </div>
+  );
+};
```
Comment (jstrzesz) [inline client/src/components/Chats.jsx:21]: add line before and after line 22

**Review by bsbrionesOS — APPROVED** [review]

**Review by Evtaft22 — APPROVED** [review]

**Pull request merged**